### PR TITLE
{lib}[GCCcore/10.3.0] nlohmann_json v3.10.0

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/b1e871ea09ebebb2cb50e8a899f341ab for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/n/nlohmann_json/nlohmann_json-3.10.0-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
+
+name = 'nlohmann_json'
+version = '3.10.0'
+
+homepage = 'https://github.com/nlohmann/json'
+description = """UShER is now a package consisting of a family of programs for 
+rapid phylogenetic analyses, particularly suitable for the SARS-CoV-2 genomes."""
```
Comment (branfosj) [inline easybuild/easyconfigs/n/nlohmann_json/nlohmann_json-3.10.0-GCCcore-10.3.0.eb:8]:
```suggestion
description = """JSON for Modern C++"""
```

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/16d0ba8d711d15ed53579813c635dbd3 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/c94dbabbf9dca82ecd43e24772d5c9e6 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @Micket!

**Pull request merged**